README.md

```diff
@@ -1,7 +1,7 @@
 # Unity-SDK
 Your plug &amp; play CMP for Unity.
 
-<mark>**Note**: Sourcepoint's Unity SDK can be used for both Android OS and iOS. When you are ready to compile your project for iOS, you will need to perform additional steps. [Click here](https://github.com/SourcePointUSA/unity-sdk#usage--build-for-ios) for more information.
+<mark>**Note**: Sourcepoint's Unity SDK can be used for both Android OS and iOS. It does nothing in Unity Editor due to architecture of SDK (it embeds native SDKs which can be accessed only when application is built for corresponding platform)
```

Comment:
I'd rephrase it like:

>  Sourcepoint's Unity SDK can be integrated with both Android and iOS. Since it embeds native SDKs, and those only work in their respective platforms, the Unity SDK can't be demoed using Unity's Editor.